pyproject.toml

```diff
@@ -18,18 +18,19 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.11.4"
```

Comment:
Not sure if that was intended. It means the library now requires Python `3.11.4` or newer and won't run on `3.9` or `3.10`.

Reply:
That wasn't intentional, no need to be so strict on this.